test-e2e/har/test_data/recording1.har

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
Is the idea here to update and check this file in every time we make changes to ajs-core?